app/src/constants.js

```diff
@@ -63,6 +68,9 @@ const Constants = {
 
   ROUTE_FILE: 'File',
 
+  ITEM_CREATE_A_CHANNEL: 'Create a channel...',
```

Comment:
Does this file contain magic constants, presentation labels, or both? Presentation labels don't generally need to be constants. 

(It looks like this one is being used as both)

Reply:
Both. I am performing some string comparison on the labels, which is why I made them constant.